### DISCUSSION TITLE
Fix BuildCraft, Project Red missing texture particles

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#186] [Bugfix Suggestions]
https://github.com/BuildCraft/BuildCraft/issues/4521
https://github.com/MrTJP/ProjectRed/issues/1400

These are a frequent reminder for how the 1.12 versions of these mods got less love.

**Comment by Primitive-Human** [comment]
That's more of a resource pack fix, should be able to add those by changing the blockstate.json for those blocks.

**Reply by xJon** [reply]
@Primitive-Human Whatever fix works!
